pkg/controller/inferenceservice/controller_test.go

```diff
@@ -125,8 +125,13 @@ func TestInferenceServiceWithOnlyPredictor(t *testing.T) {
 	mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0"})
 	g.Expect(err).NotTo(gomega.HaveOccurred())
 	c = mgr.GetClient()
-
-	recFn, requests := SetupTestReconcile(newReconciler(mgr))
+	recorder := mgr.GetEventRecorderFor(fmt.Sprintf("InferenceReconciler"))
```

Comment:
should call `InferenceServiceEventRecorder` ? also `fmt.Sprintf` is not needed here